.github/workflows/ci-cd-pull-request.yml

```diff
@@ -32,4 +32,6 @@ jobs:
         with:
           context: . # required to respect .dockerignore
           cache-from: type=registry,ref=${{ env.BASE_IMAGE }}:latest
+          build-args: |
+            ENVIRONMENT=staging
```

Comment:
Hmm, should it? I don't think it matters because this workflow doesn't deploy the built app anywhere, it's just to test that it builds successfully. But I feel like it should more closely match the workflow that runs once a PR is merged, which deploys to staging. It is only set to `production` in `ci-cd-production.yml`. Let me know if you still think it's wrong!